### ISSUE TITLE
Problems Importing SEGA CD Images (Snatcher, Sonic CD)

### ISSUE THREAD

**Issue by johnnybockmist** [jasarien/Provenance#462]
Unfortunately, I am having troubles importing SEGA CD games to Provenance 1.2.7 on iOS 10.2 (iPhone 6s).

After transferring the necessary BIOS files to Provenance, I was able to successfully transfer Final Fight CD as a zipped bin/cue file following the instructions. It was successfully imported as a SEGA CD game by Provenance and plays just fine.

However, I am not able to make Provenance import Snatcher or Sonic CD properly:

- When transferring the zipped files for Snatcher, the app loses connection and I am thrown to the home screen on iOS after the upload is finished via the browser on my PC. Nevertheless, the app is still running. When I open Provenance again, it imports the .bin file as a Genesis ROM and the .cue sheet as a SEGA CD image - none of them playable.

- When transferring Sonic CD everything seems to run just fine but again the .bin file is imported as a Genesis ROM and the .cue sheet is imported as a SEGA CD image - none of them playable.

Is there any solution to this problem?

**Comment by jasarien** [comment]
Check that the .CUE and .BIN files are named exactly the same (except for the extension, obviously). Also open the .CUE in a text editor and ensure that the .BIN file it's referencing has the correct name too.

If you could provide the crash logs for  the crash when importing Snatcher it would help to diagnose that issue.

I've definitely been able to import and play Sonic CD in Provenance (it's one of the test ROMs I used when implementing Sega CD support). Check that the ROM you have works in another Sega CD emulator, like OpenEmu, also see if you can try a different copy of the ROM (ie. a different dump), it's possible it could be a corrupted ROM dump.

**Comment by johnnybockmist** [comment]
Thanks a lot for the advice and especially for pointing me towards naming and referencing! It were indeed two different problems about the naming and referencing of the files. For Snatcher the reference in the .cue file was not exactly corresponding to the name of the .bin file. For Sonic CD the reason interestingly seemed to be that the file endings weren't spelled ".bin" and ".cue" but rather ".BIN" and ".CUE" in capital letters. I found this odd but changing the endings actually solved the problem.

I initially thought the problems were caused by Provenance, because for some reason Genesis Plus GX on my Wii didn't bother at all about those issues and played the files just fine.
